charts/evm-stack/Chart.yaml

```diff
@@ -29,10 +29,10 @@ dependencies:
     version: 0.1.0
     repository: "file://../evm-faucet"
     condition: evm-faucet.enabled
-  - name: bridge-withdrawer
+  - name: evm-bridge-withdrawer
```

Comment:
Our overrides were over the wrong chart sha in tests. The name here wasn't following norms, the name here should match the chart name, fixed the chart instead of just changing values overrides.